src/components/Navbar/NavBar.styles.js

```diff
@@ -1,6 +1,13 @@
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 
+// If you change the height of this element, go to:
```

Comment:
wouldn't it be better to just put the header height in the theme? this kind of comments shouldn't be necessary